package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-context-hook",
-  "version": "2.2.0",
+  "version": "3.0.0",
```

Comment:
The version is auto-generated by the command `yarn release`, so please leave it as is.

Reply:
Yep I did this